docker/Dockerfile

```diff
@@ -47,6 +47,9 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
     rm -rf /var/lib/apt/lists/* && \
     apt-get clean
 
+COPY init.sh /tmp/init.sh
+RUN chmod +x /tmp/init.sh && /bin/bash /tmp/init.sh
```

Comment:
I would move that whole script inline as a single run using `&&` to keep docker layers to minimum.